cup_of_join.py

```diff
@@ -0,0 +1,30 @@
+def join(*lists, sep="-"):
```

Comment:
I would use documantation instead, comments make it look messy
https://swimm.io/blog/code-comments-vs-documentation/

There's no need for the blank lines

Reply:
I fixed it